publish-sdk

```diff
@@ -85,10 +85,11 @@ update_image() {
       --repository-name "${repo}" \
       --image-ids imageTag="${VERSION}" \
       --region "us-east-1"
-  elif [[ "${REGISTRY}" =~ ^[0-9]+\.dkr\.ecr\.[a-z0-9-]+\.amazonaws\.com$ ]]; then
+  elif [[ "${REGISTRY}" =~ ^[0-9]+\.dkr\.ecr\.([a-z0-9-]+)\.amazonaws\.com$ ]]; then
     aws ecr batch-delete-image \
       --repository-name "${repo}" \
-      --image-ids imageTag="${VERSION}"
+      --image-ids imageTag="${VERSION}" \
+      --region "${BASH_REMATCH[1]}"
```

Comment:
`BASH_REMATCH` :exploding_head:  I'm continuously baffled by this language.

Reply:
That was new to me too. 😄